src/snowflake/snowpark/column.py

```diff
@@ -81,7 +81,7 @@ def _to_col_if_lit(
         return col
     elif isinstance(col, VALID_PYTHON_TYPES_FOR_LITERAL_VALUE):
         return Column(Literal(col))
-    else:
+    else:  # pragma: no cover
```

Comment:
Frankly I think these `else` cases should be fairly simple to test in a unit test, though given they are "private" functions, it is probably okay to skip.